src/utilities/PostgresWatch.ts

```diff
@@ -0,0 +1,269 @@
+     * @param {string} user_id Discord user ID
+     * @param {string} course class to alert for
+     * @param {string} channel_id id of Discord channel to send in
+     */
+    export async function insertClass(user_id: string, course: string, channel_id: string): Promise<void> {
+        try {
+            const res = await pool.query("INSERT INTO rubot.watch VALUES ($1, $2, $3, $4);",
+                [user_id, course, channel_id, false]);
+
+            GeneralUtilities.log(res, insertClass.name, "INFO");
+        }
+        catch (err) {
+            GeneralUtilities.log(err, insertClass.name, "ERROR");
+        }
+    }
+
+    /**
+     * Update a row's previously_open col in Postgres
+     * @param {string} user_id Discord user ID
+     * @param {string} course class to alert for
+     * @param {boolean} open if class should be set to open or not
+     */
+    export async function updatePrevOpenCol(user_id: string, course: string, open: boolean): Promise<unknown[]> {
+        try {
+            const res = await pool.query(`UPDATE rubot.watch SET previously_open = $3
+                                          WHERE user_id = $1 AND course = $2;`, [user_id, course, open]);
+            GeneralUtilities.log(res, dropCourseTable.name, "INFO");
+
+            return res.rows;
+        }
+        catch (err) {
+            GeneralUtilities.log(err, dropCourseTable.name, "ERROR");
+        }
+        
+        return [];
+    }
+
+    /**
+     * Drops course table if it exists
+     */
+    export async function dropCourseTable(): Promise<void> {
+        try {
+            const res = await pool.query("DROP TABLE IF EXISTS rubot.watch;");
+            GeneralUtilities.log(res, dropCourseTable.name, "INFO");
+        }
+        catch (err) {
+            GeneralUtilities.log(err, dropCourseTable.name, "ERROR");
+        }
+    }
+
+    /**
+     * Deletes a course from a user
+     * @param {string} user_id user_id of the person wanting to delete
+     * @param {string} course course to remove
+     * @returns {Promise<QueryResult>} The information about the request
+     */
+    export async function deleteRow(user_id: string, course: string): Promise<QueryResult> {
+        try {
+            const res = await pool.query(`DELETE FROM rubot.watch 
+                                          WHERE user_id = $1 AND course = $2;`, [user_id, course]);
+            GeneralUtilities.log(res, deleteRow.name, "INFO");
+
+            return res;
+        }
+        catch (err) {
+            GeneralUtilities.log(err, deleteRow.name, "ERROR");
+        }
+        return {
+            command: "DELETE",
+            rowCount: 1,
+            oid: 0,
+            rows: [],
+            fields: [],
+        };
+    }
+
+    /**
+     * Every minute, search for alerts that need to be sent now. Send if found. 
+     */
+    export function loop() {
+        // call function every minute
+        const seconds = 60;
```

Comment:
Define the constant outside of this function. Use the naming convention for constants (e.g., `const SECONDS: number = 60;`)